control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -3566,6 +3592,9 @@ components:
           description: Blobstore cluster size for the pool.
           type: integer
           format: int64
+        maxExpansion:
```

Comment:
maxExpansionSize (given clusterSize) or maxSize

Reply:
Its not in size necessarily. It can be factor too. like 20x, 30x

Reply:
hmm it'd be nice if we could show it as an actual size somewhere